resources/assets/coffee/react/beatmap-discussions/post.coffee

```diff
@@ -125,12 +125,40 @@ class BeatmapDiscussions.Post extends React.PureComponent
     @setState editing: true, =>
       @textarea.focus()
 
+  discussionLinkify: (text) =>
+    route = laroute.route('beatmapsets.discussion', {'beatmapset': "([0-9]+)"}) # what could possibly go wrong?
+    routeRegex = new RegExp(route.replace('/' , '\/'), 'i')
```

Comment:
`_.escapeRegExp`?

Reply:
except not ( ﾟ◡ﾟ)

Reply:
is anyone supposed to be able to understand these kinda of comments? because i for one cannot.

Reply:
`_.escapeRegexp` doesn't work in this context because only the forward slashes are to be escaped. nanaya realised this and exclaimed "oh right" in person when he realised (after posting the comment). Thus I added a note so to remember it had been addressed.

I was also expecting this to not have taken over a day to get merged so I didn't bother spending 5mins to write up explaining what had happened.